package.json

```diff
@@ -5,16 +5,17 @@
   "workspaces": [
     "packages/*",
     "packages/*/example",
-    "scripts/*"
+    "scripts/*",
+    "examples/*"
```

Comment:
I added the example to the workspaces so that it uses the locally built `@replayio/playwright` and `@replayio/replay` packages by default.